conduct.md

```diff
@@ -88,13 +90,15 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
+- Always be transparent on what are we doing wrong with. The reason being is because to create a mutual relationship environment between the team to understand the good and the bad of the activity together.
+
 - TODO
 - TODO
 
 #### Unacceptable Conduct
 
-- TODO
+- Never create unnecessary mistakes (Forget to git pull, never creating pull requests). This is very important to not create more problems and not to halt on the team's progress on the work.
```

Comment:
As this is an inevitability given that many students are still taking intro classes and learning about the git commands, perhaps you could restructure this to be about the lack of communication from a student IF they do make these mistakes.

Reply:
Was written by another team that already merged to main--content was pulled in when branch was created